File: Restaurant-System/src/app/Models/reservationDto.ts
import { MatDateFormats } from '@angular/material';

export interface ReservationDto {
    id: string
    reservationTime: string
    reservationDate: string
    tableSize: number
    userId: string
    userFirstName: string
    userEmail: string
    amountSpent: string
    isCheckedIn: boolean
    userPriorityStatus: string
}

File: Rest-Backend/Rest-Backend/Models/ReservationDto.cs
using Rest_Backend.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Rest_Backend.Dtos
{
    public class ReservationDto
    {
        public string Id { get; set; }
        public string ReservationTime { get; set; }
        public string ReservationDate { get; set; }
        public int TableSize { get; set; }
        public string UserId { get; set; }
        public string UserFirstName { get; set; }
        public string UserEmail { get; set; }
        public string UserPriorityStatus { get; set; }
        public string AmountSpent { get; set; }
        public bool isCheckedIn { get; set; }
    }
}
